lib/modules/manager/terragrunt/modules.ts

```diff
@@ -62,6 +62,11 @@ export function analyseTerragruntModule(
     const moduleParts = source.split('//')[0].split('/');
     if (moduleParts[0] === '..') {
       dep.skipReason = 'local';
+    } else if (source.startsWith('tfr:///')) {
+      dep.registryUrls = ['registry.terraform.io'];
```

Comment:
```suggestion
      dep.registryUrls = ['https://registry.terraform.io'];
```
shouldn't it have a https prefix?

isn't this the default registry url of terraform module datasource? so completely remove?